Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -894,3 +877,109 @@ function Invoke-ActionWithDefaultRetries
 
     Invoke-ActionWithRetries @parameters
 }
+
+function Test-OldSdk
+{
+    $sdkVersionString = (Get-SfSdkVersion)
+    if ($sdkVersionString -match "^\d+\.\d+")
+    {
+        $sdkVersion = New-Object Version
+        if ([Version]::TryParse($matches[0], [ref]$sdkVersion))
```

Comment:
Is this not good enough? Why do we need to do a match

Reply:
this will match just the major and minor version.

Reply:
removed match